src/main/java/org/team1540/robot2018/commands/auto/sequences/SwitchDoubleCube.java

```diff
@@ -0,0 +1,71 @@
+package org.team1540.robot2018.commands.auto.sequences;
+
+import static org.team1540.robot2018.commands.wrist.CalibrateWrist.CalibratePosition.OUT;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.TimedCommand;
+import org.team1540.base.util.SimpleCommand;
+import org.team1540.robot2018.Robot;
+import org.team1540.robot2018.Tuning;
+import org.team1540.robot2018.commands.arms.DropCube;
+import org.team1540.robot2018.commands.elevator.MoveElevator;
+import org.team1540.robot2018.commands.intake.AutoIntake;
+import org.team1540.robot2018.commands.intake.Eject;
+import org.team1540.robot2018.commands.wrist.CalibrateWristMP;
+import org.team1540.robot2018.motion.FollowProfile;
+
+public class SwitchDoubleCube extends CommandGroup {
+  public SwitchDoubleCube(String side) {
```

Comment:
This works, but I feel like there's gotta be a better way of doing this hierarchy from an end user ish standpoint.

Reply:
we are lowkey the end user tho

Reply:
Point is that we want things to be easier to work with.